ajax/libs/ant-design-vue/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "ant-design-vue",
+  "filename": "antd.min.js",
+  "version": "1.3.8",
+  "description": "An enterprise-class UI design language and Vue-based implementation",
+  "homepage": "https://vue.ant.design/",
```

```suggestion
  "homepage": "https://www.antdv.com",
```

Homepage appears to have migrated to a new domain.